v2/pages/issuer/opportunities/index.jsx

```diff
@@ -0,0 +1,35 @@
+import Router from 'next/router';
+import Link from 'next/link';
+import { useCollectionOnce } from 'react-firebase-hooks/firestore';
+import { firestore } from '../../../api/firebase';
+import { routes } from '../../../constants';
+
+export default () => {
+  /*
+   * TODO: Only fetch opportunities of specific issuer
+   */
+
+  const [value, loading, error] = useCollectionOnce(firestore.collection('Opportunities'));
+  return (
+    <>
+      <Link href={routes.issuer.CREATE_OPPORTUNITY}>Create new opportunity</Link>
+      {value &&
+        value.docs.map((doc) => {
+          const { title } = doc.data();
+          const { id } = doc;
+          return (
+            !loading && (
+              <div
+                key={id}
+                onClick={() =>
+                  Router.push(routes.issuer.OPPORTUNITY, `${routes.issuer.OPPORTUNITIES}/${id}`)
+                }
+              >
+                {title}
+              </div>
```

Comment:
If you'd like to make a visual element interactable, make sure you take accessibility into account.
See https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-static-element-interactions.md

It's a good idea to use a `button` container instead of a `div` as it will take care of many a11y concerns for you - such as tab indexes and keyboard listeners to click.

I like to have a global `button-container` class that looks like this:
```
.button-container {
  color: inherit;
  cursor: pointer;
  background-color: inherit;
  border: none;
  outline: none;
  padding: 0;
  margin: 0;
  text-transform: inherit;
}
```
It will preserve browser defaults for assistive technology and you can proceed to style it however you'd like in the specific component you implement it.